src/test/test-bpf-foreign-programs.c

```diff
@@ -280,9 +280,6 @@ int main(int argc, char *argv[]) {
         if (detect_container() > 0)
                 return log_tests_skipped("test-bpf fails inside LXC and Docker containers: https://github.com/systemd/systemd/issues/9666");
 
-        if (getuid() != 0)
```

Comment:
Adding the previous commit is fine, but please just leave this too, as it's doesn't hurt. Debugging these issues is slow and disruptive.